.gitlab-ci.yml

```diff
@@ -10,6 +10,7 @@ stages:
   - stats_results
   - parse_results
   - before_tests
+  - k8s_lib_injection
```

Comment:
Probably this is not going to be the final pipeline for the nightly build. Making the matrix for all the languages, we are going to launch near to 300 runners

Reply:
300 is fine :) we use 450 for end-to-end/parametric

Reply:
I can see time to time errors preparing the runners. But anyway the pipeline will be the target for the next iteration/pr